osu.Android/OsuGameActivity.cs

```diff
@@ -12,10 +12,14 @@ namespace osu.Android
     [Activity(Theme = "@android:style/Theme.NoTitleBar", MainLauncher = true, ScreenOrientation = ScreenOrientation.FullUser, SupportsPictureInPicture = false, ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize, HardwareAccelerated = false)]
     public class OsuGameActivity : AndroidGameActivity
     {
+        internal static Activity Activity { get; private set; }
```

Comment:
Just occurred to me that this `static` could potentially be dispensed with if the activity was passed to `OsuGameAndroid` via ctor, which then would cache the activity to whoever needs to use it. I'd probably prefer that from a code quality standpoint, if feasible.